viz_scripts/survey_responses.ipynb

```diff
@@ -0,0 +1,343 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ce0dcc9f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "year = 2024\n",
+    "month = 4\n",
+    "program = \"default\"\n",
+    "study_type = \"study\"\n",
+    "mode_of_interest = 'e-bike'\n",
+    "include_test_users = False\n",
+    "dynamic_labels = {}\n",
+    "use_imperial = False\n",
+    "survey_info =  {}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "585410e0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from collections import defaultdict\n",
+    "import urllib.request\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "from xml.dom import minidom\n",
+    "\n",
+    "from emcommon.survey import conditional_surveys\n",
```

Comment:
our coding convention for packages is `import XXX as YYY` and then using it as `YYY.abc`
I know we aren't currently doing that for `plots`, but we should fix that later as well.

Reply:
handled in [b02c1de](https://github.com/e-mission/em-public-dashboard/pull/124/commits/b02c1de1e19c852ae7231cc6a981763697715ed5)

Reply:
handling of `plots` import still TODO, but that seems like it should be in a cleanup and not here